cypress/e2e/test/Search_spec.cy.js

```diff
@@ -0,0 +1,27 @@
+describe("Search Functionality", () => {
```

Comment:
Hi @monikasingh949 , Can you please refactor this test same as you did the login one?